src/main/java/duke/command/Ui.java

```diff
@@ -0,0 +1,112 @@
+package duke.command;
+
+import duke.task.Task;
+
+import java.util.List;
+
+/**
+ * Dealing with responding user's input on the command line.
+ */
+public class Ui {
+    private TaskList taskList;
+    private static String logo = " ____        _        \n"
```

Comment:
I would recommend
```suggestion
    private static final String LOGO = " ____        _        \n"
```
although this line is given.